cpp/include/cudf/io/functions.hpp

```diff
@@ -238,6 +239,70 @@ struct read_csv_args {
 table_with_metadata read_csv(read_csv_args const& args,
                              rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+/**
+ * @brief Settings to use for `write_csv()`
```

Comment:
It would be helpful to have an explanation of the parameters here.

Reply:
other than `@ingroup io_writers` neither of the other writers provide any information. For consistency I've added `@ingroup io_writers`